src/traces/funnelarea/plot.js

```diff
@@ -78,10 +80,10 @@ module.exports = function plot(gd, cdModule) {
                 slicePath.attr('d', shape);
 
                 // add text
-                piePlot.formatSliceLabel(gd, pt, cd0);
+                formatSliceLabel(gd, pt, cd0);
                 var textPosition = pieHelpers.castOption(trace.textposition, pt.pts);
                 var sliceTextGroup = sliceTop.selectAll('g.slicetext')
-                    .data(pt.text && (textPosition !== 'none') ? [0] : []);
+                    .data(pt.text && (textPosition !== 'none') ? [pt] : []);
```

Comment:
Why did `[0]` become `[pt]` ?

Reply:
To have access to text `tranform`.

Reply:
Where? You have

```
sliceTextGroup.each(function() {
  // ...
})
```

 below, not extracting the `pt` object.

Reply:
See https://github.com/plotly/plotly.js/pull/4420/files#r357786862

Reply:
Cross-referencing https://github.com/plotly/plotly.js/pull/4420#discussion_r357814274